sdk/identity/identity/src/msal/nodeFlows/msalNodeCommon.ts

```diff
@@ -157,8 +157,11 @@ export abstract class MsalNode extends MsalBaseUtilities implements MsalFlow {
       auth: {
         clientId,
         authority,
-        knownAuthorities: getKnownAuthorities(tenantId, authority),
+        knownAuthorities: getKnownAuthorities(tenantId, authority, options.knownAuthorities),
         clientCapabilities,
+        authorityMetadata: options.authorityMetadata,
```

Comment:
Are these then used in MSAL directly?  I don't see any other use of them.

Reply:
Yes it is sent to MSAL directly